examples/misc_exporter_gltf.html

```diff
@@ -466,6 +500,7 @@
 				h.add( params, 'exportModel' ).name( 'Export Model' );
 				h.add( params, 'exportObjects' ).name( 'Export Sphere With Grid' );
 				h.add( params, 'exportSceneObject' ).name( 'Export Scene 1 and Object' );
+				h.add( params, 'exportCompressedObject' ).name( 'Export Coffeemat (compressed)' );
```

Comment:
nit: we might confuse people by saying "compressed" here, as the exported result no longer has Draco or Meshopt compression. Perhaps "Export Coffeemat with KTX2"?

Reply:
Replaced with "from compressed data" - otherwise I think it's also confusing because it sounds like KTX2 is exported.